src/modules/common/Nav.jsx

```diff
@@ -9,48 +9,66 @@ export default class Nav extends React.Component {
 	};
 
 	render() {
-		const NavContainer = styled.nav`
-			text-align: center;
+		const Nav = styled.nav`
+			margin: 14px;
+			position: relative;
+			text-align: left;
+			z-index: 5;
 
 			a {
-				padding: 0 5px;
+				color: white;
+				position: relative;
+				display: inline-block;
+				font-size: 1.5rem;
+				font-weight: 700;
+				margin: 8px;
+				position: relative;
+
+				&:hover {
+					text-decoration: underline;
+				}
+
+				&:nth-child(2) {
+					float: right;
+				}
 			}
 		`;
 
-		return <NavContainer>{this._renderLinks()}</NavContainer>;
+		return(
+			<Nav>
+				{this._renderLinks()}
+			</Nav>
+		)
+
 	}
 
 	_renderLinks() {
-		return [...this._standardLinks(), ...this._authLinks()];
+		return this.props.user ? this._loggedInLinks() : this._loggedOutLinks()
```

Comment:
Low Priority:
I have a pretty standard rant about `renderSomething` methods.

I pretty uniformly think they are a bad idea. You use a `renderSomething` method when you've determined that there's too much logic for this component to handle all in the `render` method. So you create some helper renderers, so that all these smaller methods can depend on the parent components `props`. Pretty soon, this component has evolved into a very large, very complicated class that takes a huge load of props. But no one wants to refactor it because they don't know which of the `renderSomething` methods maybe depends on it.

This is the crux of the issue. You have all of these mini components (the `renderSomething`s) that _implicitly_ depend on some subset of the parents `props`. _implicit_ is a bad word in programming, we want all our dependencies to be explicit.

Luckily React provides a very clean way to do this: components! An encapsulated module that takes an explicit set of data inputs and generates a piece of UI is exactly what a component is. So whenever you reach for `renderSomething`, just define a small component instead.

```jsx
function LoggedOutLinks() {
  return (
    <>
        ...
    </>
  )
}
```

Small components >> small methods in a large component. Think about how scary it is when you open up someone else's code and see a 200 line component. Think about how bite sized seeing 15 20 line components feels.

Reply:
I brought the 'nav' into it's own module. I'm going to push forward with this merge for now but would like to work with you to learn how to properly architect a separation into smaller components